releasenotes/notes/feat-dynamic-instrumentation-4c46bb86482438c1.yaml

```diff
@@ -0,0 +1,10 @@
+---
+prelude: >
+    Dynamic instrumentation allows instrumenting a running service dynamically
+    to extract runtime information that could be useful for, e.g., debugging
+    purposes, or to add extra metrics without having to make code changes and
+    re-deploy the service.
+features:
+  - |
+    Introduced the public interface for the dynamic instrumentation service. See
```

Comment:
Can we refer here to the "debugger" service to be consistent with the debugger interface that is being made public here?